x/concentrated-liquidity/types/errors.go

```diff
@@ -807,3 +807,27 @@ type CoinLengthError struct {
 func (e CoinLengthError) Error() string {
 	return fmt.Sprintf("coin length (%d) must be less than or equal to max length (%d)", e.Length, e.MaxLength)
 }
+
+type SqrtRootCalculationError struct {
+	SqrtPriceLimit sdk.Dec
+}
+
+func (e SqrtRootCalculationError) Error() string {
+	return fmt.Sprintf("issue calculating square root of price limit %s", e.SqrtPriceLimit)
+}
+
+type UninitilizedTickError struct {
+	NextTick sdk.Int
+}
+
+func (e UninitilizedTickError) Error() string {
+	return fmt.Sprintf("there are no more ticks initialized to fill the swap %s", e.NextTick)
+}
```

Comment:
This doesn't need a custom error. This can just be "there are no more ticks initialized to fill the swap" since there is no extra custom data needed to be known